src/Toolkit/Toolkit/UI/Controls/FeatureForm/RadioButtonsFormInputView.cs

```diff
@@ -115,7 +130,7 @@ private void UpdateSelection()
                 var selection = input.CodedValues.Where(a => object.Equals(a.Code, Element?.Value)).FirstOrDefault();
                 if (selection is null && input.NoValueOption == FormInputNoValueOption.Show)
                 {
-#if MAUI
+#if MAUI || WINDOWS_XAML
                     SelectedItem = null;
```

Comment:
Selecting the "no value" radio button doesn't seem to work on first click -- it becomes unchecked.
![2024-08-02_005218 Toolkit SampleApp WinUI](https://github.com/user-attachments/assets/3a0402a9-5075-49f5-9249-e6262852cf2c)

This is different from WPF, where "no value" radio button options works on the same map.